public/locales/en-US/send.ftl

```diff
@@ -162,15 +161,18 @@ signInLearnMore = Learn more!
 downloadProgressButton = Downloading... { $progress }
 downloadMessage2 = Firefox Send lets you share files with a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
 signInEmailEnter = Enter your Email
-emailEntryPlaceholder = Email
+emailEntryPlaceholder = Enter your email
+signInSizeBump = Sign in to send up to {$size}
 signInContinueMessage = to continue to Firefox Send
 signInContinueButton = Continue
 signInMenuOption = Sign in/up
+signInNextOption = Continue
 accountMenuOption = Firefox Account
-accountBenefitTitle = With a free Firefox Account with Send you can:
-accountBenefitLargeFiles = Upload larger files (up to { $size })
-accountBenefitExpiry = Have more expiry options
-accountBenefitSync = Manage your uploads across devices
+accountBenefitTitle = Create a Firefox Account or sign in to:
```

Comment:
New IDs all over the section

Reply:
@flodolo have those been merged into master and localized yet?

Reply:
@flodolo yeah just verified these are all only in the vnext branch and haven't gone to master yet. I updated the others, but these do not need ID changes since they have not been shown to localizers yet

Reply:
Sorry, had no idea this was still work against vnext. Ignore the comments